cobalt/build/linux/package.py

```diff
@@ -2,23 +2,25 @@
 """Packages Cobalt for Linux."""
 
 import os
-import shutil
 from cobalt.build import packaging
 
 
 def lay_out(out_dir, base_dir):
-  place_in_base_dir = [
-      'gen/build_info.json',
+  files = [
+      'chromedriver',
       'cobalt',
       'content_shell.pak',
       'icudtl.dat',
-      'libEGL.so',
```

Comment:
Unsure if we should keep or remove those - do gLinux builds rely on them ? Keeping them has low cost - perhaps only confusion down the line if they are or are not used ?